tests/beman/execution26/CMakeLists.txt

```diff
@@ -129,7 +129,7 @@ foreach(test ${execution_tests})
   add_test(NAME ${TEST_EXE} COMMAND $<TARGET_FILE:${TEST_EXE}>)
 endforeach()
 
-if(NOT PROJECT_IS_TOP_LEVEL)
+if(ROJECT_IS_TOP_LEVEL)
```

Comment:
Why do you change the logic?

Reply:
This change I committed by accident: during development this one check take by far most time for me:

- buildung and testing everything else takes < 2s
- this test alone takes > 10s

That interferes with my development which compiles/tests often. Sadly, I haven't found a way to disable this test without modifying the `CMakeLists.txt`. I guess, I should set up a pre-commit check making sure I undo the change. Being able to disable this test without modifying the `CMakeLists.txt` would be better, though.

Reply:
see https://github.com/bemanproject/execution26/pull/121